tests/rpmlint/rpmlint.sh

```diff
@@ -7,13 +7,15 @@ rlJournalStart
 		rlRun "tmp=\$(mktemp -d)" 0 "Create tmp directory"
 		rlRun "pushd $tmp"
 		rlRun "set -o pipefail"
+		rlIsCentOS && rlLogWarning "rpmlint on CentOS+Epel is outdated. Reported information might not be relevant"
```

Comment:
Can't we use a newer version from a different source or is this the expected behaviour?

Reply:
For `rpminspect`, probably yes, but it would require the caller to set a `context`, but I don't think `tmt` supports propagating context yet, only `environment`. We could check on the plan side if it is centos and set container manually

Reply:
I am not sure if I follow. I meant if we can't get `rpmlint` from a Copr repo or somewhere else to not need to put there a warning. (And we can filter CentOS already, can't we?)

Reply:
Oh, good point. Yeah, we can do that

I was thinking about when we make this more accessible to `Fedora-CI`, where they explicitly use a common container so that the version of `rpminspect`/`rpmlint` is common. I can see it useful to make that as an option as well.

Reply:
Aha, ok. Good point to share the versions of the tooling. But let's merge as is for now.